src/pages/Books.js

```diff
@@ -0,0 +1,39 @@
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'The Lord of the Rings',
+      author: 'J. R. R. Tolkien',
+      category: 'Fantasy',
+    },
+    {
+      id: 2,
+      title: 'Among the Thugs',
+      author: 'Bill Buford',
+      category: 'Journalism',
+    },
+    {
+      id: 3,
+      title: 'Influence',
+      author: 'Robert Cialdini',
+      category: 'Psychology',
+    },
+  ];
+
+  return (
+    <div className="book-section">
+      {books.map((book) => (
+        <div key={book.id} className="book">
+          <div className="book-info">
+            <div className="book-category">{book.category}</div>
+            <div className="book-title">{book.title}</div>
+            <div className="book-author">{book.author}</div>
+          </div>
+        </div>
+      ))}
+      <div className="gray-line" />
+    </div>
+  );
```

Comment:
- [ ] you need to add removing books the button wouldn't remove the books yet, this is just a placeholder for now

Reply:
Done.

Reply:
- [ ] I can't find the remove button still under the book as a placeholder please add that